src/condor_q.V6/queue.cpp

```diff
@@ -4318,9 +4449,25 @@ const char * const jobHold_PrintFormat = "SELECT\n"
 "   ProcId        AS ' '    NOPREFIX WIDTH 3 PRINTF '%-3d'\n"
 "   Owner         AS  OWNER WIDTH -14 PRINTAS OWNER OR ??\n"
 "   EnteredCurrentStatus  AS HELD_SINCE WIDTH 11 PRINTAS QDATE OR ??\n"
+"   HoldReasonCode AS CODE NOSUFFIX PRINTF %4d OR ?\n"
+"   HoldReasonSubCode AS '/SUB' NOPREFIX PRINTF '/%-3d' TRUNCATE OR ' '\n"
 "   HoldReason           AS HOLD_REASON WIDTH 0\n"
 "SUMMARY STANDARD\n";
 
+const char * const jobHoldCodes_PrintFormat = "SELECT\n"
+ATTR_OWNER               " AS  OWNER WIDTH -14 PRINTAS OWNER OR ??\n"
+ATTR_HOLD_REASON_CODE    " AS CODE   NOSUFFIX PRINTF %4d OR ?\n"
+ATTR_HOLD_REASON_SUBCODE " AS '/SUB' NOPREFIX PRINTF '/%-3d' OR ' '\n"
+ATTR_JOB_STATUS          " AS '   COUNT ' PRINTF %9d OR _\n"
```

Comment:
This line doesn't make sense to me... how are going from job status to counts?

Reply:
Discussed in person: ATTR_JOB_STATUS is being used as the column requires an integral value that exists. However, the reduce_holds() function overwrites the value with the count of matching job records condensed into this row of data